Makefile

```diff
@@ -98,6 +98,16 @@ endif
 build: ensure-submodule-initialized grpc/buf build-module
 	cd viam-cartographer && cmake -Bbuild -G Ninja ${EXTRA_CMAKE_FLAGS} && cmake --build build
 
+# Ideally build-asan would be added to build-debug, but can't yet 
+# as these options they fail on arm64 linux. This is b/c that 
+# platform currently uses gcc as opposed to clang & gcc doesn't
+# support using asan in this way (more work would be needed to get it to work there).
+# Ticket: https://viam.atlassian.net/browse/RSDK-3538 is the ticket to 
+# add full asan support
+build-asan: EXTRA_CMAKE_FLAGS += -DCMAKE_CXXFLAGS="-fno-omit-frame-pointer -fsanitize=address -fsanitize-address-use-after-scope -O1" -DCMAKE_EXE_LINKER_FLAGS="-fsanitize=address"
```

Comment:
@EshaMaharishi  Sorry but I can't find an example of how to assign 2 make variables on one line. 

Just adding a semicolon between the two assignments just makes the first variable contain the semicolon & the text of the other variable.

Sorry its ugly

I checked:
https://ftp.gnu.org/old-gnu/Manuals/make-3.79.1/html_chapter/make_6.html
and 
https://earthly.dev/blog/makefile-variables/

Reply:
Thanks for investigating; skimmed those two and also found this which indeed says a target can be declared multiple times to add to the list of dependencies: https://stackoverflow.com/a/43719512